CHANGELOG/CHANGELOG-1.13.md

```diff
@@ -0,0 +1,88 @@
+# Changelog 1.13
+
+## 1.13.0 - 2023-07-24
+
+### Added
+
+- Add AdminNetworkPolicy support in Antrea to align with K8s NetworkPolicy API, and document the introduction and usage. ([#5170](https://github.com/antrea-io/antrea/pull/5170) [#5270](https://github.com/antrea-io/antrea/pull/5270), [@Dyanngg])
+- Support DSR mode for Service's external addresses including LoadBalancerIPs and ExternalIPs on AntreaProxy. ([#5202](https://github.com/antrea-io/antrea/pull/5202) [#5251](https://github.com/antrea-io/antrea/pull/5251), [@tnqn])
```

Comment:
```suggestion
- Support DSR mode for Service's external addresses, including LoadBalancerIPs and ExternalIPs, on AntreaProxy. ([#5202](https://github.com/antrea-io/antrea/pull/5202) [#5251](https://github.com/antrea-io/antrea/pull/5251), [@tnqn])
```

Reply:
Refined to `Support DSR mode for Service's external addresses on AntreaProxy, including LoadBalancerIPs and ExternalIPs.`

Reply:
All comments are addressed, @tnqn @hongliangl PTAL. Thanks.